ArcadiaParties.Data.Abstractions/DTOs/DetailedUserFromAssistantDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ArcadiaParties.Data.Abstractions.DTOs
+{
+    public class DetailedUserFromAssistantDTO
+    {
+        public string employeeId { get; set; }
```

Comment:
Properties should be in PascalCase, i.e. first letter should be capitalized.

Reply:
Changed